packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2641,7 +2650,7 @@ export class ContainerRuntime
 		// If there are stashed blobs in the pending state, we need to delay
 		// propagation of the "connected" event until we have uploaded them to
 		// ensure we don't submit ops referencing a blob that has not been uploaded
-		const connecting = connected && !this._connected;
+		const connecting = connected && !this.canSendOps;
```

Comment:
Am I right that the parameter `connected` is actually more like `canSendOps`?  Can we rename it to that and add a doc comment?  Basically, fully deconstruct the "lie" from before, and just tell it like it is.  I'd even update the param name in `IRuntime`, and the comment in `Container.setContextConnectedState`.  (Renaming a param shouldn't be a breaking change)

Reply:
Oh I see you did rename it for "Core" below. So yeah this is just about pushing the truth up higher.  @kian-thompson any hesitation on the rename I'm suggesting?